manifests/params.pp

```diff
@@ -283,6 +283,26 @@
           $maxpoll         = undef
           $disable_monitor = false
         }
+        'Amazon': {
+          $config          = $default_config
+          $keys_file       = $default_keys_file
+          $driftfile       = $default_driftfile
+          $package_name    = $default_package_name
+          $service_name    = $default_service_name
+          $restrict        = [
+            'default kod nomodify notrap nopeer noquery',
+            '-6 default kod nomodify notrap nopeer noquery',
+            '127.0.0.1',
+            '-6 ::1',
+          ]
+          $iburst_enable   = false
+          $servers         = [
+            '0.centos.pool.ntp.org',
+            '1.centos.pool.ntp.org',
+            '2.centos.pool.ntp.org',
+          ]
+          $maxpoll         = undef
```

Comment:
Yeah - that was refactored on June 15th - 2 days after my last change.
Why does it take so long for any movement on this pull request. There will always be work to do if we wait for work to happen around it...